worker/include/RTC/SenderBandwidthEstimator.hpp

```diff
@@ -6,6 +6,7 @@
 #include "RTC/RateCalculator.hpp"
 #include "RTC/SeqManager.hpp"
 #include "RTC/TrendCalculator.hpp"
+#include <absl/container/btree_map.h>
```

Comment:
Why is this included her if not used in this .hpp file? If just used in the .cpp file then it must be included there.

Reply:
It is used in the .hpp file, but before was taking it from `RtpPacket.hpp`.